src/guide/computed.md

```diff
@@ -157,10 +157,10 @@ For example:
 ```
 
 ```html
-<!-- Since there is already a rich ecosystem of ajax libraries    -->
-<!-- and collections of general-purpose utility methods, Vue core -->
-<!-- is able to remain small by not reinventing them. This also   -->
-<!-- gives you the freedom to use what you're familiar with.      -->
+<!-- Dado que ya existe un rico ecosistema de bibliotecas ajax         -->
+<!-- y colecciones de métodos de utilidad de uso general, Vue core     -->
+<!-- es capaz de permanecer pequeño al no reinventarlos. Esto también  -->
+<!-- te da la libertad de usar aquello con lo que estás familiarizado. -->
 <script src="https://cdn.jsdelivr.net/npm/axios@0.12.0/dist/axios.min.js"></script>
 <script>
   const watchExampleVM = Vue.createApp({
```

Comment:
I think we should translate the messages as well here, but I'm not 100% sure. @sxcamacho ?